sdk/core/azure-core/src/test/java/com/azure/core/http/rest/SwaggerMethodParserTests.java

```diff
@@ -331,7 +331,7 @@ public void querySubstitution(Method method, Object[] arguments, Map<String, Str
             Arguments.of(substitution, null, null),
             Arguments.of(substitution, toObjectArray("raw", true), createExpectedParameters("raw", true)),
             Arguments.of(substitution, toObjectArray(null, true), createExpectedParameters(null, true)),
-            Arguments.of(substitution, toObjectArray("{sub1}", false), createExpectedParameters("%7bsub1%7d", false)),
+            Arguments.of(substitution, toObjectArray("{sub1}", false), createExpectedParameters("%7Bsub1%7D", false)),
```

Comment:
This is a semantic breaking change for users as it now encodes strings differently and requires minor version bump.

Reply:
I believe hex characters should be invariant on casing, I just copied the hex array creation from another of our classes and it used uppercase.